main/unbound/unbound.initd

```diff
@@ -22,36 +25,20 @@ depend() {
 	after auth-dns
 }
 
-checkconfig() {
-	checkpath --directory /var/run/unbound
-	UNBOUND_PIDFILE=$("${UNBOUND_CHECKCONF}" -o pidfile "${UNBOUND_CONFFILE}")
-	return $?
-}
-
 configtest() {
-	ebegin "Checking ${SVCNAME} configuration"
-	checkconfig
+	ebegin "Checking $RC_SVCNAME configuration"
+	pidfile=$("$UNBOUND_CHECKCONF" -o pidfile "$UNBOUND_CONFFILE")
+	checkpath --directory /run/unbound
 	eend $?
 }
 
-start() {
-	checkconfig || return $?
-	ebegin "Starting ${SVCNAME}"
-	start-stop-daemon --start --pidfile "${UNBOUND_PIDFILE}" \
-		--exec "${UNBOUND_BINARY}" -- -c "${UNBOUND_CONFFILE}"
-	eend $?
-}
-
-stop() {
-	checkconfig || return $?
-	ebegin "Stopping ${SVCNAME}"
-	start-stop-daemon --stop --pidfile "${UNBOUND_PIDFILE}"
-	eend $?
+start_pre() {
+	configtest || return $?
 }
 
 reload() {
-	checkconfig || return $?
-	ebegin "Reloading ${SVCNAME}"
-	start-stop-daemon --signal HUP --pidfile "${UNBOUND_PIDFILE}"
+	configtest || return $?
+	ebegin "Reloading $RC_SVCNAME"
+	kill -SIGHUP $(head -n 1 $pidfile)
```

Comment:
Why have you replaced `start-stop-daemon --signal HUP --pidfile "$pidfile"` with `kill -SIGHUP $(head -n 1 "$pidfile")`?